src/test/java/htsjdk/variant/vcf/VCFPathReaderTest.java

```diff
@@ -0,0 +1,63 @@
+package htsjdk.variant.vcf;
+
+import com.google.common.jimfs.Configuration;
+import com.google.common.jimfs.Jimfs;
+import htsjdk.HtsjdkTest;
+import htsjdk.tribble.TestUtils;
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Test;
+
+import java.nio.file.FileSystem;
+import java.nio.file.Path;
+
+/**
+ * Created by farjoun on 10/12/17.
+ */
+public class VCFPathReaderTest extends HtsjdkTest {
```

Comment:
Now that you unified the classes, this should be named `VCFFileReaderTest`

Reply:
thanks.